ci-operator/config/stackrox/collector/stackrox-collector-release-3.11.yaml

```diff
@@ -0,0 +1,555 @@
+    rhel8-1-drivers:
+      as:
+      - replaced-by-osci:rhel8-1-drivers
+    rhel8-2-drivers:
+      as:
+      - replaced-by-osci:rhel8-2-drivers
+    rhel8-3-drivers:
+      as:
+      - replaced-by-osci:rhel8-3-drivers
+    scripts:
+      as:
+      - replaced-by-osci:scripts
+  to: drivers
+resources:
+  '*':
+    limits:
+      memory: 20Gi
+    requests:
+      cpu: "2"
+      memory: 2Gi
+tests:
+- as: merge-lint
+  postsubmit: true
+  steps:
+    test:
+    - as: clang-format
+      commands: make -C collector check
+      from: src
+      resources:
+        requests:
+          cpu: 2000m
+          memory: 4000Mi
+    - as: shfmt-check
+      commands: make shfmt-check
+      from: src
+      resources:
+        requests:
+          cpu: 2000m
+          memory: 4000Mi
+- as: merge-test-scripts
+  postsubmit: true
+  steps:
+    test:
+    - as: shellcheck-all
+      commands: make shellcheck-all
+      from: src
+      resources:
+        requests:
+          cpu: 2000m
+          memory: 4000Mi
+- as: merge-unit-tests
+  postsubmit: true
+  steps:
+    test:
+    - as: collector-unit-tests
+      commands: /go/src/github.com/stackrox/collector/cmake-build/collector/runUnitTests
+      from: collector-binary
+      resources:
+        requests:
+          cpu: 2000m
+          memory: 4000Mi
+- as: merge-push-images
+  postsubmit: true
+  steps:
+    test:
+    - as: push-collector-images
+      commands: .openshift-ci/jobs/push-images/push-images.sh
+      credentials:
+      - mount_path: /tmp/secret/stackrox-stackrox-push
+        name: stackrox-stackrox-push
+        namespace: test-credentials
+      dependencies:
+      - env: COLLECTOR_SLIM
+        name: collector-slim
+      - env: COLLECTOR_FULL
+        name: collector-full
+      from: src
+      resources:
+        requests:
+          cpu: 2000m
+          memory: 4000Mi
+- as: merge-rhel-7-ebpf-integration-tests
+  postsubmit: true
+  steps:
+    env:
+      COLLECTION_METHOD: ebpf
+      IMAGE_FAMILY: rhel-7
+      VM_TYPE: rhel
+    workflow: stackrox-collector-integration-tests
+- as: merge-rhel-7-kernel-module-integration-tests
+  postsubmit: true
+  steps:
+    env:
+      COLLECTION_METHOD: kernel-module
+      IMAGE_FAMILY: rhel-7
+      VM_TYPE: rhel
+    workflow: stackrox-collector-integration-tests
+- as: merge-rhel-8-ebpf-integration-tests
+  postsubmit: true
+  steps:
+    env:
+      COLLECTION_METHOD: ebpf
+      IMAGE_FAMILY: rhel-8
+      VM_TYPE: rhel
+    workflow: stackrox-collector-integration-tests
+- as: merge-rhel-8-kernel-module-integration-tests
+  postsubmit: true
+  steps:
+    env:
+      COLLECTION_METHOD: kernel-module
+      IMAGE_FAMILY: rhel-8
+      VM_TYPE: rhel
+    workflow: stackrox-collector-integration-tests
+- as: merge-cos-85-lts-ebpf-integration-tests
+  postsubmit: true
+  steps:
+    env:
+      COLLECTION_METHOD: ebpf
+      IMAGE_FAMILY: cos-85-lts
+      VM_TYPE: cos
+    workflow: stackrox-collector-integration-tests
```

Comment:
COS 85 has been removed because it is no longer available in GCP, I think we can go ahead and remove it here as well, since it will just cause a know flake.

Reference: #32078

Reply:
Done